airflow/jobs.py

```diff
@@ -224,15 +224,50 @@ def reset_state_for_orphaned_tasks(self, dag_run, session=None):
         # also consider running as the state might not have changed in the db yet
         running = self.executor.running
         tis = list()
-        tis.extend(dag_run.get_task_instances(state=State.SCHEDULED, session=session))
-        tis.extend(dag_run.get_task_instances(state=State.QUEUED, session=session))
+        tis.extend(dag_run.get_task_instances(state=[State.SCHEDULED, State.QUEUED], session=session))
```

Comment:
Actually, I don't think we need to reset state for SCHEDULED. SCHEDULED will be picked up in "_execute_helper", or am I misreading that?